integ/test_cloudwatch/windows-config/fluent-bit.conf

```diff
@@ -0,0 +1,52 @@
+[SERVICE]
+    Log_Level debug
+
+[INPUT]
+    Name              forward
+    Listen            0.0.0.0
+    Port              24542
```

Comment:
the standard fluent port is 24224, why not use that?

Reply:
So that port can be used by the customers to send logs to FLB. Therefore, we are using another port for sending the traffic usually sent on Linux via Unix Sockets.
This would prevent any port contention.

Reply:
> So that port can be used by the customers to send logs to FLB.

Sorry I don't understand at all- this is an integ test, customers are not sending any logs to the test.... we send the logs and have full power to determine the port I think?

Reply:
Yes that is correct. We can switch to 24224 as well. Given that the proposal for Firelens is to use dedicated port 24542, I had used it here.
If you prefer that we use 24224, I can change it the next revision.